director/director_api.go

```diff
@@ -232,19 +232,19 @@ func ConfigFilterdServers() {
 		for _, sn := range param.Director_FilteredServers.GetStringSlice() {
 			filteredServers[sn] = permFiltered
 		}
-		log.Debugln("Loaded filtered servers config from Director.FilteredServers param", filteredServers)
+		log.Debugln("Loaded server downtime configuration from the Director.FilteredServers parameter:", filteredServers)
 	}
 
-	if param.Director_DbLocation.IsSet() {
-		persistedServerDowntimes, err := GetAllServerDowntimes()
+	if param.Director_DbLocation.GetString() != "" {
+		persistedServerDowntimes, err := getAllServerDowntimes()
 		if err != nil {
-			log.Error("Failed to read persisted ServerDowntimes from director db", err)
+			log.Error("Failed to read persisted server downtimes from director db", err)
```

Comment:
This log line should have a colon to separate the two halves of the error message.

Reply:
gotcha. thanks!